src/components/RoomView/GameStatus.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import GuessesRemaining from './GuessesRemaining';
+import { useState } from 'react';
+const GameStatus = () => {
+  let gameStatus = useSelector((state) => state.game.status);
+  const { playerId } = useSelector((state) => state.player);
+  const { teamOneOperatives, teamOneSpymaster } = useSelector((state) => state.teamOne);
+  const { teamTwoOperatives, teamTwoSpymaster } = useSelector((state) => state.teamTwo);
+  const [playerNote, setPlayerNote] = useState('');
+
+  const joinTeamNote = 'Join a team to play the game';
```

Comment:
eslint says this is never used

Reply:
i think we will use this once we are fine tuning the details of what we want on the UI before a user joins a team, but just putting on pause for now. will make a note in file!